lib/init.coffee

```diff
@@ -85,9 +85,10 @@ module.exports =
 
     @_log 'providing linter, examining javac-callability.'
 
+    name: 'linter-javac'
```

Comment:
Usually the name provided here is the name of the source program, `javac` in this case.